modules/system-health-dashboard-details.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+:_module-type: CONCEPT
+[id="system-health-dashboard-details_{context}"]
+= System health dashboard details
+
+To access the health dashboard:
+
+* On the RHACS portal, navigate to *Platform Configuration* -> *System Health*.
+
+The health dashboard organizes information in the following groups:
+
+* *Cluster Health* - Shows the overall state of {product-title} cluster.
+* *Vulnerability Definitions* - Shows the last update time of vulnerability definitions.
+* *Image Integrations* - Shows the health of all registries that you have integrated.
+* *Notifier Integrations*  - Shows the health of any notifiers (Slack, email, Jira, or other similar integrations) that you have integrated.
+* *Backup Integrations* - Shows the health of any backup providers that you have integrated.
+
+The dashboard lists the following states for different components:
+
+* *Healthy* - The component is functional.
+* *Degraded* - The component is partially unhealthy.
+This state means the cluster is functional, but some components are unhealthy and require attention.
+* *Unhealthy* - This component is not healthy and requires immediate attention.
+* *Uninitialized* - The component has not yet reported back to Central to have its health assessed.
+An uninitialized state may sometimes require attention, but often components report back the health status after a few minutes or when the integration is used.
+
+[discrete]
+== Cluster health section
+
+The *Cluster Overview* shows information about your {product-title} cluster health.
+It reports the health information about the following:
+
+* *Collector Status* - It shows whether the Collector pod, which the {product-title} is using is reporting healthy.
```

Comment:
which -> that, and needs some other tweaking. Possibly:

"It shows whether the Collector pod that {product-title} uses is reporting healthy."

Same format for the rest of this list.